archinfo/arch.py

```diff
@@ -374,12 +374,14 @@ def _get_register_dict(self):
                 res[reg] = (r.vex_offset + offset, size)
         return res
 
-    @property
-    def bytes(self):
-        """
-        The standard word size in bytes, calculated from the ``bits`` field
-        """
-        return self.bits // self.byte_width
+    def __getattr__(self, a):
+        if a == 'bytes':
+            # The standard word size in bytes, calculated from the ``bits`` field
+            v = self.bits // self.byte_width
+        else:
+            raise AttributeError(a)
+        setattr(self, a, v)
```

Comment:
Should we hook `__setattr__` of `bits/byte_width` so that they don't desync if any madman ever changes them after? If not why don't we just store the attribute directly? Then we don't even need the `__getattr__`